### ISSUE TITLE
OctoPack not passing --format to octo.exe

### ISSUE THREAD

**Issue by jjschlesinger** [cake-build/cake#1349]
I am running 0.17 of cake.

OctoPack will always create a .nupkg even when setting Zip as the format...

```csharp

OctoPack("MyPackadeId", new OctopusPackSettings 
	{
		Format = OctopusPackFormat.Zip,
		Overwrite = true
	});
```

Here's the output...

```
Executing: "./tools/OctopusTools/tools/Octo.exe" pack --id MyPackageId --overwrite
```

I also checked OctopusDeployPacker.cs, and the --format switch isn't being appended to the builder.

**Comment by patriksvensson** [comment]
Ouch. Not super familiar with the Octo tool wrappers but has this worked before?

**Comment by jjschlesinger** [comment]
I'm not sure. This is the first time I have tried to create a zip file.

**Comment by patriksvensson** [comment]
Ok, just wanted to check if it was a regression. Either way we will need to fix this. Thanks for reporting it!

Are you up for sending a PR? :smile:

**Comment by jjschlesinger** [comment]
Sure, I'll do a PR.

**Comment by devlead** [comment]
Fixed by  #1350
